.github/workflows/ci.yaml

```diff
@@ -11,3 +11,17 @@ on:
 jobs:
   nox:
     uses: ./.github/workflows/reusable-nox.yml
+
+  check:
+    if: always()
+
+    needs:
+      - nox
+
+    runs-on: Ubuntu-latest
```

Comment:
```suggestion
    runs-on: ubuntu-latest
```

Reply:
@oraNod do you want to commit this from the UI and merge right away?

Reply:
@webknjaz Sure. I can also do it now.